tools/topology/topology2/CMakeLists.txt

```diff
@@ -17,4 +17,8 @@ else()
 	endif()
 endif()
 
+if (CONFIG_CAVS)
 add_subdirectory(cavs)
+elseif(CONFIG_ACE)
+add_subdirectory(ace)
+endif()
```

Comment:
As already mentioned by @keqiaozhang , this does not make sense to me. We've always built all topologies all the time; it's fairly quick and there is no toolchain or platform dependency. I suggest reverting the entire change in this file.

Reply:
How about we remove checking on CONFIG_ACE and go ahead to create both subdirectories `cavs` and `ace`?

We previously aligned that we want to separate IPC4 topology binaries for MTL and cAVS platforms:
For MTL+ platform release: /lib/firmware/intel/sof-ace-tplg 
For internal testing on cAVS platforms: /lib/firmware/intel/avs-tplg